wagtail/api/v2/serializers.py

```diff
@@ -16,7 +16,7 @@ class TypeField(Field):
     Serializes the "type" field of each object.
 
     Example:
-    "type": "wagtailimages.Image"
+    "type": "media.Image"
```

Comment:
Since `wagtailimages.Image` is still the default image model unless using the project template (or overriding in a custom project template), I think it still makes sense to use that here.